ci/Makefile

```
@@ -0,0 +1,8 @@
+
+
+set-autoscaler-pipeline:
+	@./autoscaler/set-pipeline.sh
+
+unpause-pipeline:
+	@./scripts/unpause-pipeline.sh
+
```

Fine for me to have a `Make`-file for this but let's declare the targets as phony

```suggestion
.PHONY: set-autoscaler-pipeline
set-autoscaler-pipeline:
	@./autoscaler/set-pipeline.sh

.PHONY: unpause-pipeline
unpause-pipeline:
	@./scripts/unpause-pipeline.sh

```